app/controllers/partnering_organizations_controller.rb

```diff
@@ -17,6 +17,12 @@ def index
     end
   end
 
+ #GET /partnering_organization/subscriptions
+ def subscriptions
```

Comment:
So, this actually isn't necessary because we're passing the `@partnering_organization.subscribers` directly into the react component in the `subscriptions.html.erb` file